src/app/services/constraint.service.ts

```diff
@@ -180,7 +193,7 @@ export class ConstraintService {
     if (hasUnselected) {
       let result: CombinationConstraint = new CombinationConstraint();
       result.combinationState = CombinationState.Or;
-      result.mark = ConstraintMark.OBSERVATION;
+      result.dimension = Constraint.TOP_LEVEL_DIMENSION;
```

Comment:
shouldn't the dimension attribute of a combination constraint default to TOP_LEVEL_DIMENSION? which also means removing TOP_LEVEL_DIMENSION from Constraint class